src/Libraries/CoreNodes/Thread.cs

```diff
@@ -8,15 +8,15 @@ public static class Thread
         /// <summary>
         ///     Pauses the current evaluation thread for a given amount of time.
         /// </summary>
-        /// <param name="x">Object to pass through.</param>
+        /// <param name="object">Object to pass through.</param>
         /// <param name="msTimeout">
         ///     Amount of time to pause the thread, in milliseconds.
         /// </param>
-        /// <returns name="x">Object passed through.</returns>
-        public static object Pause(object x, int msTimeout)
+        /// <returns name="object">Object passed through.</returns>
+        public static object Pause(object @object, int msTimeout)
```

Comment:
why are you using `@` in front of `object`?

Reply:
when I used VS to automatically rename x to object it added the @ symbol, I don't know if this was to avoid a problem it is called the same as the class object, should I erase the @ symbol? (it didn't appear in the node)

Reply:
ah, its because `object` is a keyword so cant be use as a argument name, apparently adding `@` is like an escape character that allow keywords to be used as names. Its probably fine if thats the only way we can rename the input.